cli/fossilize_replay_linux.hpp

```diff
@@ -1430,15 +1430,6 @@ static int run_slave_process(const VulkanDevice::Options &opts,
                              const ThreadedReplayer::Options &replayer_opts,
                              const vector<const char *> &databases)
 {
-	// Just in case the driver crashed due to stack overflow,
```

Comment:
Any particular reason for this code shuffle? The thread_callback in ThreadedReplayer will call sigaltstack() and expects this struct to be filled in.